internal/eval/eval_test.go

```diff
@@ -1442,3 +1410,26 @@ func testEvalHSET(t *testing.T, store *dstore.Store) {
 
 	runEvalTests(t, tests, evalHSET, store)
 }
+
+func testEvalSELECT(t *testing.T, store *dstore.Store) {
+	tests := map[string]evalTestCase{
+		"nil value": {
+			setup:  func() {},
+			input:  nil,
+			output: []byte("-ERR wrong number of arguments for 'select' command\r\n"),
+		},
+		"database is specified": {
+			setup:  func() {},
+			input:  []string{"1"},
+			output: clientio.RespOK,
+		},
+	}
+	runEvalTests(t, tests, evalSELECT, store)
+}
+
+func BenchmarkEvalSELECT(b *testing.B) {
+	store := dstore.NewStore(nil)
+	for i := 0; i < b.N; i++ {
+		evalSELECT([]string{"1"}, store)
+	}
+}
```

Comment:
We can remove this benchmark, it's not needed for this command.

Reply:
@JyotinderSingh Sure, will remove this. However, I added this since the PR description asked to do `benchmem`.